### PR TITLE
Using Github Actions instead of Travis CI (CU-86dthadwg)

### PR DESCRIPTION
**Pull request by Darrickg**
Both Github Actions and Travis are running on this branch and both (should) pass.

Potential Issue:
image
There are some "errors" when setting up the database with Github Actions but it is not detecting it as a failure and everything passes so I'm not sure if it is technically an error.

Test Plan:

- [x] Passes all test cases locally
- [x] Passes Github Actions
- [x] Passes Travis CI
- [x] Runs normally in dev

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-86dthadwg Use GitHub Actions instead of Travis CI](https://app.clickup.com/t/86dthadwg)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+          until pg_isready -h localhost -p 5433 -U brave -d bravebuttons; do
+            echo "Waiting for PostgreSQL to be ready..."
+            sleep 1
+          done
+
+      # setup database
+      - name: Set up database
+        working-directory: ./server
+        run: |
+          ./setup_postgresql_local_dev.sh
```
Comment (micahdbak) [inline .github/workflows/ci.yml:86]: I would recommend copying the commands from this file and having them execute from within the script - it should help you change what's causing the errors you're finding.

**Review by micahdbak — COMMENTED** [review]
Looking really good - once we resolve the problem with "Set up database" I approve this PR!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (micahdbak) [reply .github/workflows/ci.yml:86]: Note that this file is intended for __local development database setup__, not necessarily Travis. You'll want to setup PG for the GH environment, and then run the setup_postgresql.sh file directly

**Comment by Darrickg** [comment]
removed travis and made changes to the README (specifically on the badge and instructions on travis)

still need to make the names consistent with Elana's

**Review by micahdbak — APPROVED** [review]
Awesome job. Glad this is done. Feel free to merge :)

**Pull request merged**